logstash-core-event/spec/logstash/event_spec.rb

```diff
@@ -354,8 +359,13 @@
       end
 
       it "should warn for invalid string format" do
-        expect(LogStash::Event::LOGGER).to receive(:warn)
+        logger_mock = double("logger")
```

Comment:
I see some duplication in here? I know is not in the style of the spec, but we might introduce this now and make usage of let statements as you do for the core-event-java spec.